images/scripts/fedora.setup

```diff
@@ -3,6 +3,11 @@
 set -ex
 IMAGE="$1"
 
+# Use daily UDisks2 stack, for catching regressions early and testing
+# new APIs early.
+
+dnf copr -y enable @storage/udisks-daily
```

Comment:
Yes, let's rather do this as a scenario. We also want to cover udisks (and its dependencies) updates in Fedora proper for fedora-testing.

Reply:
Correct. So this rather needs an update to https://github.com/cockpit-project/cockpit/blob/main/test/run , and possibly renaming the scenario (in testmap and cockpit's test/run) if you care. But the change *here* is wrong -- it would apply to all Fedora images. You at least need to guard that with some `if fedora-testing`. (But a scenario is better)